css-four-column-layout/styles.css

```diff
@@ -4,47 +4,46 @@
 
 body {
   margin: 0;
+  font-family: Barlow, sans-serif;
 }
 
 h2 {
-  text-align: center;
-  font-weight: bold;
-  font-family: Barlow, sans-serif;
-  position: relative;
-  left: 30px;
+  font-size: 18px;
 }
 
-h3 {
-  font-weight: bold;
-  font-family: Barlow, sans-serif;
+.container {
+  width: 1000px;
```

Comment:
You should make your container smaller, so that you can use a `col-fifth` class to set each of your columns to a more accurate size, 10% is **_really_** small, and doesn't seem intuitive based on seeing a row of five items.